backend/activity_log/tests/test_api_views.py

```diff
@@ -58,3 +58,40 @@ def test_list_api_filter_by_event_type(self):
 
         self.assertEqual(response.data[0]["total"], 1)
         self.assertEqual(response.data[0]["date"], str(self.today.date()))
+
+
+class TestActivityLogEventTypeViewSet(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "activitylogeventtype-list"
+    SUPPRESS_TEST_GET_LIST = True
+    SUPPRESS_TEST_POST_TO_CREATE = True
+
+    def setUp(self):
+        self.today = timezone.now()
+
+        self.entry1 = factories.EventTypeFactory()
+        self.entry1 = factories.LogEntryFactory()
+        self.entry1.timestamp = self.today
+        self.entry1.save()
+
+        self.entry2 = factories.EventTypeFactory()
+        self.entry2 = factories.LogEntryFactory()
+        self.entry2.timestamp = self.today
+        self.entry2.save()
+
```

Comment:
do we not log in a generic user here to see that anyone logged in can access? or is assumed that this will be the case?